### DISCUSSION TITLE
ChatBot output of "I don't know"

### DISCUSSION THREAD

**Discussion by jbinnscornick** [FlowiseAI/Flowise#1001] [Q&A] [answered]
Hi,

When my Chatbot gets asked a question, that it cannot find an answer for in my JSON file, it outputs "I don't know" (or something like that). How can i change it so it outputs something like "Sorry, I do not have the answer to that question, Please contact support"?

Thanks,
Jacob

**Comment by chungyau97** [comment]
Hi @jbinnscornick,

You could write into the `system message` saying reply user with "Sorry, I do not have the answer to that question, Please contact support" if you do not have the answer.

**Reply by jbinnscornick** [reply]
Hi Chung,

How do i do that?

Thanks,
Jacob

**Reply by chungyau97** [reply, marked as answer]
It will be under Additional Parameters > System Message.

**Reply by jbinnscornick** [reply]
Hi Chung,

where do i find those settings?


Thanks,
Jacob
